mne/label.py

```diff
@@ -448,12 +453,38 @@ def fill(self, src, name=None):
         # pos
         pos = hemi_src['rr'][vertices]
 
-        if name is None:
-            name = self.name
+        name = self.name if name is None else name
         label = Label(vertices, pos, values, self.hemi, self.comment, name,
                       None, self.subject, self.color)
         return label
 
+    def restrict(self, src, name=None):
+        """Restrict a label to a source space.
+
+        Parameters
+        ----------
+        src : instance of SourceSpaces
+            The source spaces to use to restrict the label.
```

Comment:
name param not documented? why did the docstring test pass here?

Reply:
Good question, that must be a bug of some sort in our docstring testing code